packages/mongo-nav/src/mongo-select/OrgSelect.tsx

```diff
@@ -57,10 +57,8 @@ import {
 const orgButtonStyle = css`
   justify-content: space-between;
   border-radius: 5px 0 0 5px;
-  border: 1px solid ${uiColors.gray.light2};
```

Comment:
Am I missing something, or did we only remove a border, not add one? 🤔

Reply:
this is a dupe from here: https://github.com/mongodb/leafygreen-ui/blob/main/packages/mongo-nav/src/mongo-select/styles.ts#L45, and the line that I removed from the project select file was simply overwriting this one.

Reply:
if you run locally, you should see both borders as in the screenshot attached